File: HoneydewTestProject/ReferencesABViaNamespacePrefix.cs
using HoneydewTestProject.A;

namespace HoneydewTestProject
{
    public class ReferencesABViaNamespacePrefix
    {
        public void MethodC1A(C1 c1Params)
        {
            var x = c1Params.X;
        }

        public void MethodC1B(B.C1 c1Params)
        {
            var x = c1Params.X;
        }
    }
}

File: vb/HoneydewTestProjectVB/HoneydewTestProjectVB/ReferencesABViaNamespacePrefix.vb
Imports HoneydewTestProjectVB.A

Public Class ReferencesABViaNamespacePrefix
    Public Sub MethodC1A(ByVal c1Params As C1)
        Dim x = c1Params.X
    End Sub

    Public Sub MethodC1B(ByVal c1Params As B.C1)
        Dim x = c1Params.X
    End Sub
End Class
